operate/client/e2e-playwright/tests/processInstanceListeners.spec.ts

```diff
@@ -41,19 +42,23 @@ test.describe('Process Instance Listeners', () => {
     processInstancePage.navigateToProcessInstance({id: processInstanceKey});
 
     // Start flow node should NOT enable listeners tab
-    await page.getByLabel('StartEvent_1').click();
+    await processInstancePage.instanceHistory.getByText('StartEvent_1').click();
     await expect(processInstancePage.listenersTabButton).not.toBeVisible();
 
     // Service task flow node should enable listeners tab
-    await page.getByLabel(/service task b/i).click();
+    await processInstancePage.instanceHistory
+      .getByText(/service task b/i)
+      .click();
     await expect(processInstancePage.listenersTabButton).toBeVisible();
   });
 
   test('Listeners data displayed', async ({page, processInstancePage}) => {
     const processInstanceKey = initialData.instance.processInstanceKey;
     processInstancePage.navigateToProcessInstance({id: processInstanceKey});
 
-    await page.getByLabel(/service task b/i).click();
+    await processInstancePage.instanceHistory
+      .getByText(/service task b/i)
+      .click();
```

Comment:
@pedesen Here I just refactored to use the `instanceHistory` locator so it's more specific and less error prone. The new test in this file is what's important.